### PR TITLE
Allow user to select Fleet repo protocol select.

### PR DESCRIPTION
**Pull request by Shavindra**
### Summary
Fixes rancher/dashboard#7518

### Occurred changes and/or fixed issues
Requirement changed to display a banner instead.

### Technical notes summary
- Continuous Delivery > Git Repos > Add repository button on top right
- Banner should be displayed.

### Areas or cases that should be tested
image

### CODE REVIEW

**Comment by richard-cox** [comment]
@Shavindra The PR template isn't filled in, lint is failing and there's no reviewer requested. Can you update?

**Comment by kwwii** [comment]
We should remove the "://" from the protocol label and also write them with all caps (HTTPS and SSH).

**Comment by kwwii** [comment]
+1 if you can autodetect the type from the text entered by the user.

File: shell/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -1988,7 +1988,10 @@ fleet:
       empty: The root of the repo is used by default.  To use one or more different directories, add them here.
     repo:
       label: Repository URL
-      placeholder: 'e.g. https://github.com/rancher/fleet-examples.git'
+      protocol: Protocol
+      placeholder: 
+        ssh: e.g. git@github.com:rancher/dashboard.git
```
Comment (nwmac) [inline shell/assets/translations/en-us.yaml:1993]: Use the same fleet-examples repo, but with ssh protocol

**Review by nwmac — CHANGES_REQUESTED** [review]

**Comment by nwmac** [comment]
@Shavindra You should mark the conversations as resolved when addressed and re-request a review when ready.

**Comment by nwmac** [comment]
@Shavindra @kwwii I don't know if this is an improvement - we could have 'just' added a banner or tip saying that you can use either an https or ssh URL - we don't auto-detect the protocol with this change, so you can't just paste in a `git@` URL and have the protocol change to ssh - so you now have two steps.

I thought some of the auth options only applied to either https or ssh - but we don't seem to do anything to update those based on which protocol you select.

I think it would be better just to adda line saying https or ssh URLs can be used.

**Review by nwmac — CHANGES_REQUESTED** [review]
See comment

**Review by bisht-richa — APPROVED** [review]
LGTM

**Review by nwmac — APPROVED** [review]

**Pull request merged**